docker-database/DB_dockerDir/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM postgres:15-alpine
+
+COPY *.sql /docker-entrypoint-initdb.d/
+# why does the example have these two "equivalent" syntaxes 
```

Comment:
COPY is to be preferred for copying files from local filesystem into the image.

ADD, which is the original instruction, is to be preferred when downloading a remote file to the container and/or unzipping/untarring an archive